queries/swift/highlights.scm

```diff
@@ -1,3 +1,6 @@
+[ "." ";" ":" "," ] @punctuation.delimiter
+[ "\\(" "(" ")" "[" "]" "{" "}"] @punctuation.bracket
```

Comment:
ideally "\\("` should be `@punctuation.special` in string interpolation. but it's not easy to match against

`"?"` after optional types needs a parser change: https://github.com/alex-pinkus/tree-sitter-swift/issues/100